k8spsb/README.md

```diff
@@ -6,18 +6,18 @@ For more info visit the [Ocopea website](https://ocopea.github.io).
 
 # How to build
 
-* clone the kubernetes repo under GOPATH/src/ocopea/
-* run the `buildImage.sh` script from the k8spsb folder
+* Clone the kubernetes repo under GOPATH/src/ocopea/
+* Run the `buildImage.sh` script from the k8spsb folder
 
 # How to run
 
-k8spsb is being automatically deployed every time you run the 
-[Kubernetes deployer](https://github.com/ocopea/kubernetes)
+K8spsb is automatically deployed every time you run the 
+[Kubernetes deployer](https://github.com/ocopea/kubernetes/tree/master/deployer)
```

Comment:
Please keep the original link here, linking to the "deployer" package leads to the source code without any documentation, the root kuberentes repo is what we need here

Reply:
will do